cluster/Vagrantfile

```diff
@@ -29,8 +29,10 @@ def create_cfg_info(num_nodes, node_ips)
            else
              {}
            end
-    node['control'] = ENV['CONTIV_CONTROL_IF'] || 'eth1'
-    node['data'] = ENV['CONTIV_DATA_IF'] || 'eth2'
+    def_ctrl_if =  ENV['CONTIV_NODE_OS'] && ENV['CONTIV_NODE_OS'] == 'ubuntu' ? 'enp0s8' : 'eth1'
```

Comment:
`Hash#[]` returns `nil` when a key is not found so you can remove the first half of the boolean condition on checks like these without issue.

`nil == "any string"` is false (or more specifically `NilClass#==` just returns false if the input is a string)